server/test/integration/com/thoughtworks/go/server/service/ValueStreamMapServiceIntegrationTest.java

```diff
@@ -92,13 +89,14 @@ public void setUp() throws Exception {
         result = new HttpLocalizedOperationResult();
 
         username = new Username(new CaseInsensitiveString("admin"));
-
+        notifier.disableUpdates();
     }
 
     @After
     public void teardown() throws Exception {
         dbHelper.onTearDown();
         configHelper.onTearDown();
+        notifier.enableUpdates();
```

Comment:
Usually, that which is done at the end of `setUp` is reverted at the beginning of `teardown`. Convention, though. Not super important.